src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -710,6 +747,25 @@ export const selectPortfolioTotalFiatBalanceByAccount = createSelector(
   },
 )
 
+export const selectPortfolioAccountBalanceByAccountNumberAndChainId = createSelector(
+  selectPortfolioTotalFiatBalanceByAccount,
+  selectPortfolioAccountMetadata,
+  selectAccountNumberParamFromFilter,
+  selectChainIdParamFromFilter,
+  (accountBalances, accountMetadata, accountNumberString, chainId): string => {
+    const accountNumber = parseInt(accountNumberString.toString())
+    if (isNaN(accountNumber))
```

Comment:
Normally would comment against use of NaN, but given `parseInt` was used to parse `accountNumberString`, this is fine 🤌

Reply:
as discussed lodash `isNumber` is a better pattern for this valid zero value 🤦